content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,448 @@
+```
+
+After saving the config file switch to the terminal with your Prometheus docker container and stop it by pressing `ctrl+C` and start it again to reload the configuration by using the existing [command](#run-prometheus).
+
+The terminal should return the message `"Server is ready to receive web requests."` and after a few seconds you should start to see colourful graphs in [your Prometheus](http://localhost:9090/graph?g0.range_input=5m&g0.stacked=0&g0.expr=probe_http_duration_seconds&g0.tab=0).
+
+This works, but it has a few disadvantages:
+
+1. The actual targets are up in the param config, which is very unusual and hard to understand later.
+1. The `instance` label has the value of the blackbox exporter’s address which is technically true, but not what we are interested in.
+1. We can’t see which URL we probed. This is unpractical and will also mix up different metrics into one if we probe several URLs.
+
+To fix this, we will use [relabeling](/docs/prometheus/latest/configuration/configuration/#<relabel_config>).
+Relabeling is very powerful for advanced usecases because behind the scenes many things in Prometheus are configured with internal labels.
+The details are worthy of a [talk](https://www.youtube.com/watch?v=b5-SvvZ7AwI) on its own. For now is suffices if you understand this:
+
+* All labels starting with `__` are dropped after the scrape. Most internal labels start with `__`.
+* You can set internal labels that are called `__param_<name>`. Those set URL parameter with the key `<name>` for the scrape request.
+* There is an internal label `__address__` which is set by the `targets` under `static_configs` and whose value is the hostname for the scrape request. By default it is later used to set the value for the label `instance`, which is attached to each metric and tells you were the metrics came from.
+
+Here is the config you will use to do that. Don’t worry if this is a bit much at once, we will go through it step by step:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
+
+- job_name: 'blackbox'
+  scrape_interval: 5s
+  metrics_path: /probe
+  params:
+    module: [http_2xx]
+  static_configs:
+      - targets:
+        - http://prometheus.io    # Target to probe with http.
+        - https://prometheus.io   # Target to probe with https.
+        - http://example.com:8080 # Target to probe with http on port 8080.
+  relabel_configs:
+    - source_labels: [__address__]
+      target_label: __param_target
+    - source_labels: [__param_target]
+      target_label: instance
+    - target_label: __address__
+      replacement: localhost:9115  # The blackbox exporter’s real hostname:port. For Windows and macOS replace with - host.docker.internal:9115
+```
+
+So what is new compared to the [last config](#prometheus-config)?
```

Comment:
I like the below, but maybe split it in "results" and "what are the steps to get there ", this again subsplit for each of the three sections under `relabel_configs`?

Reply:
Good point.
I tried to change this using the suggestion feature, but github is confused by nested markdown code blocks within suggestions. I will change it directly. Sorry for the inconvenience.

Reply:
Looking good.